.copier-answers.yml

```diff
@@ -17,6 +17,7 @@ org_name: Odoo Community Association (OCA)
 org_slug: OCA
 rebel_module_groups:
 - sale_packaging_default,sale_order_product_recommendation,sale_order_product_recommendation_packaging_default,sale_order_product_recommendation_elaboration
+- sale_order_qty_change_no_recompute
```

Comment:
@pedrobaeza @victoralmau was there a specific reason why this module was not marked as rebel?

Reply:
https://github.com/OCA/sale-workflow/pull/3023